djangocms_admin_style/sass/components/_mobile.scss

```diff
@@ -46,8 +46,10 @@ div.breadcrumbs {
 // login
 body.login {
     #container {
-        width: auto !important;
+        width: 90% !important;
         max-width: 320px;
+        min-width: inherit !important;
+        margin-top: 100px;
         #content #content-main {
```

Comment:
@ruslanmalogulko thank you. Removed the whole #content styles. They are not necessary